stackslib/src/net/http.rs

```diff
@@ -725,13 +729,18 @@ impl HttpResponsePreamble {
         }
     }
 
-    pub fn ok_JSON_from_md<W: Write>(
+    fn ok_JSON_from_md<W: Write>(fd: &mut W, md: &HttpResponseMetadata) -> Result<(), codec_error> {
+        Self::ok_JSON_with_status(fd, md, 200)
+    }
+
+    fn ok_JSON_with_status<W: Write>(
```

Comment:
Strictly speaking, this isn't an "OK" if `status` is not 200.  Can you refactor this method to be called `JSON_with_status` and add `accepted_JSON_with_status()` as the helper method for rendering a 202 response?